sklearn/linear_model/_bayes.py

```diff
@@ -226,8 +244,31 @@ def fit(self, X, y, sample_weight=None):
         self : object
             Returns the instance itself.
         """
+
         self._validate_params()
 
+        max_iter = self.max_iter
+        # TODO(1.4) Remove
```

Comment:
This would be going into version `1.3` hence the `TODO` and warnings should refer to `1.3` and `1.5`. This suggestion carries for all other uses of `1.2` and `1.4` in this PR.

```suggestion
        # TODO(1.5) Remove
```

Reply:
Thanks for the suggestions. Working on it as soon as possible.